signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -21,7 +21,9 @@ def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--pose', required=True, type=str, help='path to input pose file')
     parser.add_argument('--elan', required=True, type=str, help='path to elan file')
-    parser.add_argument('--model', type=str, default='359a544.ckpt', help='model to use')
+    parser.add_argument('--model', type=str, default='bc2de71.ckpt', help='model to use')
+    parser.add_argument('--strategies', required=False, type=str, default='tight',
```

Comment:
should be `strategy`, no need for `required=False` because there is `default`, `help` can be more helpful, "segmentation strategy to use"